js/app.js

```diff
@@ -57,43 +55,80 @@ function errorNotice (message) {
 
 }
 
+function tableCreate(userList,userCount) {
+    
+
+var count = document.getElementById('countbody');
+
+if (count)
+{
+    $("#countbody").remove();
+   
+}
+
+    var tblbody = document.createElement('tbody');
+  tblbody.setAttribute('id','countbody');
```

Comment:
Here the table body `id` is used to check if it is already existing. In case of form resubmission, this needs to be cleared out.